conflation/trace_fetching/mapillary.py

```diff
@@ -198,181 +206,332 @@ def make_trace_data_requests(
-    seq_page = 1
-    while seq_next_url:
-        print("@@ MAPILLARY: Seq Page {}, url={}".format(seq_page, seq_next_url))
-        seq_resp = session_.get(seq_next_url, timeout=10)
-        seq_ids = []
-        for seq_f in seq_resp.json()["features"]:
-            seq_id = seq_f["properties"]["key"]
-
-            # If we've already processed this seq_id before, skip it, otherwise we will be writing duplicate image data
-            if seq_id in sequences_by_id:
-                print(
-                    "@@@ MAPILLARY: Skipping seq_id={} b/c we've already seen it on a previous page".format(
-                        seq_id
-                    )
-                )
-                continue
 
-            # Only process sequences that originated from this bbox. This prevents us from processing sequences twice
-            origin_lon, origin_lat = seq_f["geometry"]["coordinates"][0]
-            if not is_within_bbox(origin_lon, origin_lat, bbox_as_list):
-                print(
-                    "@@@ MAPILLARY: Skipping seq b/c origin ({}, {}) not in bbox {}".format(
-                        origin_lon, origin_lat, bbox
-                    )
-                )
-                continue
-
-            # Skip sequences that have too few images
-            if len(seq_f["geometry"]["coordinates"]) < skip_if_fewer_imgs_than:
-                continue
-
-            seq_ids.append(seq_id)
-
-        if len(seq_ids) > 0:
-            # Paginate images within these sequences
-            img_next_url = IMAGES_URL.format(access_token, ",".join(seq_ids), img_per_page)
-            img_page = 1
-            while img_next_url:
-                print("@@@ MAPILLARY: Image Page {}, url={}".format(img_page, img_next_url))
-                img_resp = session_.get(img_next_url, timeout=10)
-                for img_f in img_resp.json()["features"]:
-                    if img_f["properties"]["sequence_key"] not in sequences_by_id:
-                        sequences_by_id[img_f["properties"]["sequence_key"]] = []
-                    sequences_by_id[img_f["properties"]["sequence_key"]].append(
-                        {
-                            "time": parser.isoparse(
-                                img_f["properties"]["captured_at"]
-                            ).timestamp(),  # Epoch time
-                            "lon": img_f["geometry"]["coordinates"][0],
-                            "lat": img_f["geometry"]["coordinates"][1],
-                        }
+    # We will use this dict to group trace points by sequence ID
+    sequences = []
+    # We will use this set to skip sequences we've already processed
+    seen_sequences = set()
+
+    resp = session_.get(
+        COVERAGE_TILES_URL.format(BBOX_SECTION_ZOOM, tile[0], tile[1], access_token)
+    )
+
+    tile_pb = vector_tile_pb2.Tile()
+    tile_pb.ParseFromString(resp.content)
+
+    for layer in tile_pb.layers:
```

Comment:
The code is somewhat self explanatory in this parsing, basically pulls out the data we need from the proto, then does the same checks as we used to do.